keps/sig-release/create-conformance-doc.md

```diff
@@ -0,0 +1,87 @@
+---
+kep-number: draft-20190121
+title: Provide Conformance Document as part of the release docs for each Kubernetes minor release
+authors:
+  - "@brahmaroutu"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+  - sig-docs
+  - wg-conformance
+reviewers:
+  - @spiffxp
+  - @timothysc
+  - @ixdy
+  - @sig-docs-en-reviews
+  - @zacharysarah
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-01-21
+last-updated: 2019-01-21
+status: provisional
+---
+
+# Provide Conformance Document as part of the release docs for each Kubernetes minor release
+
+## Table of Contents
+
+- [Conformance Document as part of the release docs](#breaking-apart-the-kubernetes-test-tarball)
+  - [Table of Contents](#table-of-contents)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Graduation Criteria](#graduation-criteria)
+  - [References](#references)
+  - [Implementation History](#implementation-history)
+
+## Summary
+
+The Kubernetes release artifacts must include conformance document as part of release documents.
+
+This KEP proposes to add docs/conformance folder with a single conformance document that will 
+contain list of all conformance tests that are part of this release.
+file: docs/conformance/conformance.md 
+
+
+## Motivation
+
+As there is no proper place to publish conformance document per release, currently
+conformance document is checked in under the repo cncf/k8s-conformance. Even though
+this is a useful location but each release specific document should be part of the
+kubernetes release process. 
+If we can automatically build conformance document as part of each minor release it 
+should be shipped along with the release tar that contains other documentation for
+the release.
+
+### Goals
+
+* Build process needs to be updated to create a conformance binary that can generate conformance document
```

Comment:
I'm not sure what the process is now but it looks like it may just be based on the comments on the tests? What kind of checks are there to make sure those are accurate?

I actually didn't know these documents existed and have had to just generate my own list using `ginkgo ... dryRun` to find the tests in question.

Reply:
@johnSchnake conformance binary was originally built to list all conformance tests and then enhanced to generate documentation based on the comments added in front of the test. Yes there is no tool to validate the comments but the review process is intensive enough when you promote a e2e test to conformance test.

Reply:
👍 I'll have to look more into what already exists I suppose. Good to know about this documentation though; I've had multiple people just try to find such a list and all I could do was point them to how to run the dryRun tests. This will be great to pass on and get it more visible.